### DISCUSSION TITLE
Resolution 1280 x 720 , cannot be added

### DISCUSSION THREAD

**Discussion by Alkanino** [elementary/triage#441] [Q&A]
I have an AMD Ryzen 5 laptop, with an ati/radeon card AMD Radeon graphics, compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Lucienne (rev c2), the 1270x720 resolution does not appear, I have tried xrander to add it but it does not appear In the monitor resolution list, could you help me what the configuration would be for said video card.

**Comment by truefakts** [comment]
This sounds more like a xrandr specific question than an elementary OS question. Adding a custom resolution would be the same process across linux distributions that utilize randr and X.org. Checkout this post on the Ubuntu forums https://askubuntu.com/questions/377937/how-do-i-set-a-custom-resolution

Out of curiously, what is your use case for a 1270x720 resolution? 1280x720 would be typical for 720p. Is that just a typo, or do you have an exotic monitor?
